evalml/pipelines/components/transformers/preprocessing/featuretools.py

```diff
@@ -28,6 +28,8 @@ def __init__(self, index="index", features=None, random_seed=0, **kwargs):
         self.index = index
         self.features = features
         self._passed_in_features = True if features else None
+        # If features are passed in, they'll have a dataframe_name we should utilize
+        self._dataframe_name = self.features[0].dataframe_name if self.features else "X"
```

Comment:
The use of `self.features[0]` here assumes that all features were created from the same dataframe.

I didn't add a comment here explaining that, since it seems that assumption is true for the whole component (for example, in how we create the entityset in `_make_entity_set`), but I'd be happy to note that here if it would be useful.

Reply:
let's add the note!